.github/labeler.yml

```diff
@@ -0,0 +1,108 @@
+# Documentation - https://github.com/actions/labeler
+
+# General Labels
+'Build | Project System':
+  - '.github/*'
+  - '.github/**/*'
+  - '*.sln'
+  - '**/*.sln'
+  - '*.vcxproj*'
+  - '**/*.vcxproj*'
+  - 'cmake/*'
+  - 'cmake/**/*'
+  - 'CMakeLists.txt'
+  - '**/CMakeLists.txt'
+  - 'buildbot.xml'
+  - 'build.sh'
+'Dependencies':
+  - '3rdparty/*'
+  - '3rdparty/**/*'
+  - '**/3rdpartyDeps.props'
+  - '.gitmodules'
+'Documentation':
+  - '*.md'
+  - '**/*.md'
+  - '*.pdf'
+  - '**/*.pdf'
+'GUI/WX':
+  - 'pcsx2/gui/*'
+  - 'pcsx2/gui/**/*'
+'GameDB':
+  - '**/GameIndex.*'
+'Installer | Package':
+  - 'nsis/*'
+  - 'nsis/**/*'
+  - 'build.sh'
+
+# Tools / Features
+'Debugger':
+  - 'pcsx2/DebugTools/*'
+  - 'pcsx2/DebugTools/**/*'
+  - 'pcsx2/gui/Debugger/*'
+  - 'pcsx2/gui/Debugger/**/*'
+'IPC':
+  - 'pcsx2/IPC*'
+  - 'pcsx2/**/IPC*'
+'TAS Functionality':
+  - 'pcsx2/Recording/*'
+  - 'pcsx2/Recording/**/*'
+
+# Emulation Components
+'Counters':
+  - 'pcsx2/Counters.*'
+'Vector Units':
+  - 'pcsx2/VU*'
+  - 'pcsx2/**/VU*'
+  - 'pcsx2/*VU*'
+  - 'pcsx2/**/*VU*'
+'VIF':
+  - 'pcsx2/Vif*'
+  - 'pcsx2/**/Vif*'
+  - 'pcsx2/VIF*'
+  - 'pcsx2/**/VIF*'
+
+# GS Related Labels
+'Plugin: GS': # TODO - update after plugin merge
+  - '**/GSdx/*'
+  - '**/GSdx/**'
+'GS: Direct3D':
```

Comment:
There's a couple of more renderer specific stuff, you can check the gsdx source folders.

Reply:
As in more files pertaining to Direct3D?  Or there being other GSdx related labels -- if so, I wasn't sure how to support them ie (Mipmapping, Upscaling, etc) because there didn't seem to be dedicated files to those features.